db/migrate/20191203001550_add_clearance_to_users.rb

```diff
@@ -10,7 +10,7 @@ def self.up
 
     add_index :users, :remember_token
 
-    users = select_all('SELECT id FROM users WHERE remember_token IS NULL')
+    users = select_all("SELECT id FROM users WHERE remember_token IS NULL")
```

Comment:
I think the quote updates are fine to leave in this PR, especially since this is an app for your own experimentation. I would like to point out that for our client repositories, we would typically make stylistic changes like this in their own PR. This results in fewer things to think about when reviewing code, and allows for better management of the repo through smaller, focused changes.

Reply:
Great. Noted, thanks! I am definitely interested in improving my workflow/commit history to follow this. Be prepared for stylized PR's coming your way 👊